common/src/main/java/com/hibiscusmc/hmccosmetics/database/types/MySQLData.java

```diff
@@ -52,8 +52,7 @@ public void setup() {
     @Override
     public void clear(UUID uniqueId) {
         Bukkit.getScheduler().runTaskAsynchronously(HMCCosmeticsPlugin.getInstance(), () -> {
-            try {
-                PreparedStatement preparedSt = preparedStatement("DELETE FROM COSMETICDATABASE WHERE UUID=?;");
+            try (PreparedStatement preparedSt = preparedStatement("DELETE FROM COSMETICDATABASE WHERE UUID=?;")) {
```

Comment:
This is not done anywhere else in the plugin. For consistency's sake, could you move this into the block where it was?

Reply:
Sure